spec/support/test_ledger/operation/full_update.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../operation'
+
+module LedgerSync
+  module Ledgers
+    module TestLedger
+      class Operation
+        class FullUpdate
+          include TestLedger::Operation::Mixin
+
+          private
+
+          def find_resource
+            client.find(
+              path: ledger_resource_path
+            )
+          end
+
+          def operate
+            find_result_hash = find_resource.body[quickbooks_online_resource_type.to_s.classify]
```

Comment:
`quickbooks_online_resource_type`
```suggestion
            find_result_hash = find_resource.body[test_ledger_resource_type.to_s.classify]
```